dbms/src/Flash/Coprocessor/CoprocessorReader.h

```diff
@@ -38,6 +39,103 @@
 #pragma GCC diagnostic pop
 
 
+namespace pingcap
+{
+namespace common
+{
+
+template <typename T>
+class CopIterMPMCQueue : public IMPMCQueue<T>
```

Comment:
What is the purpose of wrapping this class? It seems that MPMCQueue can be used directly.

Reply:
`IMPMCQueue` and `MPMCQueueResult` belongs to `client-c`.

Reply:
ok